code/src/main/java/com/onlinecontacttracing/servlets/get-data/CalendarDataForPositiveUser.java

```diff
@@ -14,28 +14,43 @@
 import java.util.List;
 import java.util.Collections;
 import java.util.Optional;
+import com.google.maps.GeoApiContext;
+import com.google.maps.PlacesApi;
+import com.google.maps.model.PlacesSearchResult;
 import com.googlecode.objectify.Objectify;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.FileInputStream;
+import java.io.File;
 import java.util.logging.Logger;
 import java.util.Set;
 import com.onlinecontacttracing.storage.PotentialContact;
+import com.onlinecontacttracing.storage.PositiveUserPlaces;
+import com.google.maps.errors.ApiException;
+import java.lang.InterruptedException;
+import java.io.IOException;
 
 class CalendarDataForPositiveUser implements Runnable {
 
   private static final String APPLICATION_NAME = "Get Calendar Data From Positive User";
   private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
   private static final String calendarType = "primary";
   static final Logger log = Logger.getLogger(CalendarDataForPositiveUser.class.getName());
+  private static final String CREDENTIALS_FILE_PATH = "WEB-INF/apiKey.txt";
+  private static final int apiKeyLength = 39;
 
   private final Objectify ofy;
   private final String userId;
   private final Credential credential;
   Set<PotentialContact> contacts;
```

Comment:
contacts is passed in from the servlet because the servlet needs to use it later on to merge contact sets. How should I make contacts not an instance and still have a pointer to it in the servlet?

Reply:
Hmmm... perhaps then let's make these changes: 1) still modify the method to return them, and then store that in the run() method to the instance field 2) add a public method called "getContacts" (rather than relying on package visibility

Reply:
Done